packages/cozy-dataproxy-lib/src/components/Search/useFetchResult.jsx

```diff
@@ -7,6 +7,31 @@ import { getIconForSearchResult } from './getIconForSearchResult'
 
 const log = Minilog('🔍 [useFetchResult]')
 
+const searchWithRetry = async (
+  dataProxy,
+  searchValue,
+  { maxRetries = 5, delay = 500 } = {}
```

Comment:
500 is maybe an high delay for the first attempt ? But I don't know what time can this kind of thing take.

Reply:
The very first attempt will be right away. It is actually the second attempt that will make 500ms. The performances measures in the PR description give some insights of the time to have initialized index

Reply:
Let's try with this and we'll see.